pegasus/sites.v3/code.org/public/css/tutorial-landing.css

```diff
@@ -42,8 +44,19 @@
   border-radius: 8px;
 }
 
+.header {
+  padding: 10px;
+}
+
+@media screen and (min-width: 513px) and (max-width: 600px) {
+  .choice {
+    padding: 10px;
+  }
+}
+
 @media screen and (min-width: 0px) and (max-width: 512px) {
   .choice {
-    width: 70%;
+    width: 100%;
```

Comment:
Did you want to make it 70% so you could see the others sooner on mobile? It looks funny so small with a lot of margin on the right. I kinda like it full size.